digitalforms-api/src/main/java/ca/bc/gov/open/digitalformsapi/viirp/utils/DigitalFormsConstants.java

```diff
@@ -9,5 +9,12 @@ public final class DigitalFormsConstants {
 	
 	// MDC constants
 	public static final String REQUEST_CORRELATION_ID = "correlationId";
+	
+	
+	// Exception constants
+	public static final String EXCEPTION_NOT_FOUND = "Not Found.";	//404
+	public static final String EXCEPTION_UNAUTHORIZED = "The requester is unauthorized."; //401
+	public static final String EXCEPTION_FORBIDDEN = "Forbidden."; //403
```

Comment:
Do we currently use these constants somewhere or are these for later?

Reply:
These are for when you use the various exception types. Keeps the output string the same as the specification for the various HTTP status codes.